step-templates/azure-devops-build-pipeline.json

```diff
@@ -0,0 +1,83 @@
+    },
+    "Parameters": [
+      {
+        "Id": "5f4bdb73-ce24-4d5d-8541-82efae04b9f5",
+        "Name": "AzureDevOpsAccessKey",
+        "Label": "Azure DevOps Access Key",
+        "HelpText": "A [personal access token (PAT)](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=preview-page) is used as an alternate password to authenticate into Azure DevOps. Learn how to create, use, modify, and revoke PATs for Azure DevOps.",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "Sensitive"
+        }
+      },
+      {
+        "Id": "63dd21d5-3d38-4bc2-9b09-9e2e4efa5974",
+        "Name": "AzureDevOpsOrganizationName",
+        "Label": "Azure DevOps Organization Name",
+        "HelpText": "The name of the Azure DevOps [organization](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/organization-management?view=azure-devops).",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "0d641c0c-bd5f-408a-9d9a-6ac710bd134c",
+        "Name": "AzureDevOpsProjectName",
+        "Label": "Azure DevOps Project Name",
+        "HelpText": "Project ID or project name",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "b5e6ea31-5679-48b6-a5ec-4c7274dbb398",
+        "Name": "AzureDevOpsPipelineName",
+        "Label": "Azure DevOps Pipeline Name",
+        "HelpText": "The name of the pipeline you want to run in your Azure DevOps project\n\nExample: helloword-ci",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "62f3eab8-9785-4b96-9c52-a0a12f794e12",
+        "Name": "AzureDevOpsBranch",
+        "Label": "Azure DevOps Branch",
+        "HelpText": "Source branch to build from",
+        "DefaultValue": "Main",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "ea425cd0-110e-45de-b38f-1c07b82cb390",
+        "Name": "AzureDevOpsWaitUntilCompletion",
+        "Label": "Azure DevOps Wait Until Completion",
+        "HelpText": "Wait for the pipeline to finish. If selected then the step will wait until the build pipeline is completed before finishing the step. If the build pipeline fails then the step will also fail.",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "Checkbox"
+        }
+      }
+    ],
+    "$Meta": {
+      "ExportedAt": "2021-03-25T13:12:50.932Z",
+      "OctopusVersion": "2020.5.5",
+      "Type": "ActionTemplate"
+    },
+    "LastModifiedBy": "adamoctoclose",
+    "Category": "Azure DevOps"
```

Comment:
@adamoctoclose - This should be `azure-devops` - this is the category it looks for the icon for (and is why the build is failing)